fineract-provider/src/main/java/org/apache/fineract/commands/api/AuditsApiResource.java

```diff
@@ -63,19 +59,15 @@
 @RequiredArgsConstructor
 public class AuditsApiResource {
 
-    private static final Set<String> RESPONSE_DATA_PARAMETERS = new HashSet<>(Arrays.asList("id", "actionName", "entityName", "resourceId",
-            "subresourceId", "maker", "madeOnDate", "checker", "checkedOnDate", "processingResult", "commandAsJson", "officeName",
-            "groupLevelName", "groupName", "clientName", "loanAccountNo", "savingsAccountNo", "clientId", "loanId", "url"));
-
     private static final String RESOURCE_NAME_FOR_PERMISSIONS = "AUDIT";
 
     private final PlatformSecurityContext context;
     private final AuditReadPlatformService auditReadPlatformService;
     private final ApiRequestParameterHelper apiRequestParameterHelper;
-    private final DefaultToApiJsonSerializer<AuditData> toApiJsonSerializer;
-    private final DefaultToApiJsonSerializer<AuditSearchData> toApiJsonSerializerSearchTemplate;
-    private final SqlValidator sqlValidator;
+    private final ToApiJsonSerializer<String> toApiJsonSerializer;
 
+    // todo: why response dto depend to query param paged..? Impossible refactor normally. Probably need to separate to
```

Comment:
You can raise a question in the Fineract DEV mail list whether people are using the List or Paged version and recommend to deprecate one of them. Personally, only the paged version should be here!